libafl_bolts/src/lib.rs

```diff
@@ -12,6 +12,7 @@
 // For `std::simd`
 #![cfg_attr(nightly, feature(portable_simd))]
 // For `core::error`
+#![cfg_attr(nightly, feature(error_in_core))]
```

Comment:
this is not needed. if you update to latest rust, it gives warnings

Reply:
Yes but if you don't update it it doesn't compile, maybe remove it in a few weeks?